docs/operator-manual/applicationset/Template.md

```diff
@@ -13,14 +13,14 @@ Here is the template subfield from a Cluster generator:
 # (...)
  template:
    metadata:
-     name: '{{cluster}}-guestbook'
+     name: '{{ .nameNormalized }}-guestbook'
```

Comment:
I believe `name` is also valid...not just `nameNormalized` ...maybe add an example of that or just use `name` as that's what most people use.

Reply:
[Yes, `.name` is also valid!](https://argo-cd.readthedocs.io/en/stable/operator-manual/applicationset/Generators-Cluster)

But I think there is no place where `.name` makes sense in these examples. Each place where `cluster` was previously used has some pre-/suffix to it (`guestbook/` or `-guestbook`). Not using the normalized name would risk ending up with something bad like `My cluster name 🙃-guestbook`, which we should not encourage.

I guess we could make a change that removes the suffix and uses `.name` appropriately, but I prefer to keep the change minimal rather than rewriting too much. And using `.name` may also risk breaking things, so we should probably run the examples first to ensure we do not introduce badness…

Do you agree? Should we keep this change minimal or rewrite the examples to include `.name`?

Reply:
Actually, good point. In this context it's best to use `nameNormalized` ...maybe a small change of a comment like

```

# (...)
 template:
   metadata:
     name: '{{cluster}}-guestbook'
     name: '{{ .nameNormalized }}-guestbook' # {{ .name }} is also valid but not recommended for this example
```

Reply:
Awesome, that LGTM!